docarray/array/mixins/match.py

```diff
@@ -50,6 +51,7 @@ def match(
             elements. When `darray` is big, this can significantly speedup the computation.
         :param exclude_self: if set, Documents in ``darray`` with same ``id`` as the left-hand values will not be
                         considered as matches.
+        :param filter: filter query used for pre-filtering or filtering
```

Comment:
```suggestion
        :param filter: filter query used for pre-filtering
```
In match we only use filter for pre-filtering right ?